z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -281,55 +281,57 @@ cd ocp_upi
+| `local_rhcos_image`       | \<local-rhcos-image-url\>                              | This is all rhcos images download [link](https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/latest/), download the name that corresponds with KVM or z/VM images, and use an SSH or HTTP client to store it, put the link here                                                                                                                                                                                                                                                                                                                               |
+| `additional_certs`        | `{{ lookup('file', '/opt/registry/certs/domain.crt') \ | indent (width=2) }}`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | The local mirror registry repo additionally need SSL certificated to be accessed, those can be added cert file via the `additional_certs` variable.
+| `create_server_timeout`   | 10                                                     | Default is 10 minutes that used to create instances and volumes from backend storage provider                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `use_internal_bastion`    | true                                                   | (Boolean) true or false, if true then nodes information under /var/named/\<clustername>\.zone and /etc/haproxy/haproxy.cfg will update and remove automatically                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
@shhull please correct me. I think `use_internal_bastion` is `required` not `optional`. It should be put together with `os_dns_domain` (line 301) In README.md.